contribs/gmf/src/filters/filterselectorcomponent.js

```diff
@@ -693,15 +704,15 @@ gmf.FilterselectorController.RuleCache;
  *     directedRules: (Array.<ngeo.rule.Rule>)
  * }}
  */
-gmf.FilterselectorController.RuleCacheItem;
+gmf.filters.filterselectorComponent.Controller_.RuleCacheItem;
 
 
-gmf.module.component('gmfFilterselector', {
+gmf.filters.filterselectorComponent.component('gmfFilterselector', {
   bindings: {
     active: '=',
     map: '<',
     toolGroup: '<'
   },
-  controller: gmf.FilterselectorController,
-  templateUrl: () => `${gmf.baseTemplateUrl}/filterselector.html`
+  controller: gmf.filters.filterselectorComponent.Controller_,
+  templateUrl: () => `${gmf.baseModuleTemplateUrl}/filterselectorcomponent.html`
```

Comment:
a `/filter` in the path is missing no ? have you tried the example and the application ?